<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.2.335">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>R/Bioconductor for Multi-Omics Data Science</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./q-and-a.html" rel="next">
<link href="./images/logo.png" rel="icon" type="image/png">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>


<meta property="og:title" content="R/Bioconductor for Multi-Omics Data Science">
<meta property="og:description" content="Learn to Analyze Genomics, Metagenomics, and Transcriptomics Data with R">
<meta property="og:site-name" content="R/Bioconductor for Multi-Omics Data Science">
<meta name="twitter:title" content="R/Bioconductor for Multi-Omics Data Science">
<meta name="twitter:description" content="Learn to Analyze Genomics, Metagenomics, and Transcriptomics Data with R">
<meta name="twitter:image" content="https://chiraltraining.github.io/WW01_DataAnalysiswithR//images/bioconductor.png">
<meta name="twitter:creator" content="@hossainlab">
<meta name="twitter:image-height" content="600">
<meta name="twitter:image-width" content="518">
<meta name="twitter:card" content="summary_large_image">
</head>

<body class="nav-sidebar docked">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
    <div class="container-fluid d-flex justify-content-between">
      <h1 class="quarto-secondary-nav-title">R/Bioconductor for Multi-Omics Data Science</h1>
      <button type="button" class="quarto-btn-toggle btn" aria-label="Show secondary navigation">
        <i class="bi bi-chevron-right"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse sidebar-navigation docked overflow-auto">
    <div class="pt-lg-2 mt-2 text-center sidebar-header">
      <a href="./index.html" class="sidebar-logo-link">
      <img src="./images/logo.png" alt="" class="sidebar-logo py-0 d-lg-inline d-none">
      </a>
      <div class="sidebar-tools-main">
    <a href="https://github.com/hossainlab/" title="GitHub" class="sidebar-tool px-1"><i class="bi bi-github"></i></a>
</div>
      </div>
      <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
      </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link active">Home</a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./q-and-a.html" class="sidebar-item-text sidebar-link">FAQs</a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./modules.html" class="sidebar-item-text sidebar-link">Modules</a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./packages.html" class="sidebar-item-text sidebar-link">Packages</a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./resources.html" class="sidebar-item-text sidebar-link">Resources</a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./license.html" class="sidebar-item-text sidebar-link">License</a>
  </div>
</li>
    </ul>
    </div>
</nav>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#overview" id="toc-overview" class="nav-link active" data-scroll-target="#overview">Overview</a>
  <ul class="collapse">
  <li><a href="#what-is-r" id="toc-what-is-r" class="nav-link" data-scroll-target="#what-is-r">What is R?</a></li>
  <li><a href="#what-is-bioconductor" id="toc-what-is-bioconductor" class="nav-link" data-scroll-target="#what-is-bioconductor">What is Bioconductor?</a></li>
  <li><a href="#learning-objectives" id="toc-learning-objectives" class="nav-link" data-scroll-target="#learning-objectives">Learning Objectives</a></li>
  <li><a href="#why-r" id="toc-why-r" class="nav-link" data-scroll-target="#why-r">Why R?</a></li>
  <li><a href="#useful-resources-for-learning-r" id="toc-useful-resources-for-learning-r" class="nav-link" data-scroll-target="#useful-resources-for-learning-r">Useful resources for learning R</a></li>
  <li><a href="#textbook" id="toc-textbook" class="nav-link" data-scroll-target="#textbook">Textbook</a></li>
  <li><a href="#required-software" id="toc-required-software" class="nav-link" data-scroll-target="#required-software">Required software</a></li>
  <li><a href="#recording-of-classes" id="toc-recording-of-classes" class="nav-link" data-scroll-target="#recording-of-classes">Recording of classes</a></li>
  <li><a href="#is-this-course-for-me" id="toc-is-this-course-for-me" class="nav-link" data-scroll-target="#is-this-course-for-me">Is this course for me?</a></li>
  <li><a href="#prework" id="toc-prework" class="nav-link" data-scroll-target="#prework">Prework</a></li>
  <li><a href="#install-bioconductor-packages" id="toc-install-bioconductor-packages" class="nav-link" data-scroll-target="#install-bioconductor-packages">Install Bioconductor Packages</a></li>
  <li><a href="#zoom-working-virtually" id="toc-zoom-working-virtually" class="nav-link" data-scroll-target="#zoom-working-virtually">Zoom + Working Virtually</a></li>
  <li><a href="#team" id="toc-team" class="nav-link" data-scroll-target="#team">Team</a></li>
  </ul></li>
  </ul>
<div class="toc-actions"><div><i class="bi bi-github"></i></div><div class="action-links"><p><a href="https://github.com/hossainlab/clinical-reporting-gtsummary-rmed/edit/main/index.qmd" class="toc-action">Edit this page</a></p><p><a href="https://github.com/hossainlab/clinical-reporting-gtsummary-rmed/issues/new" class="toc-action">Report an issue</a></p></div></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title d-none d-lg-block">R/Bioconductor for Multi-Omics Data Science</h1>
<p class="subtitle lead">Learn to Analyze Genomics, Metagenomics, and Transcriptomics Data with R</p>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<!-- https://github.com/rstudio-conf-2022/teach-ds -->
<div class="cell">
<div class="cell-output-display">
<p><img src="images/bioconductor_banner.jpg" class="img-fluid" width="1200"></p>
</div>
</div>
<hr>
<p>🗓 March 25, 2023 | 8:00am - 100:00pm BDT</p>
<p>🏨 Medium - Zoom/Google Meet</p>
<p>💥 <strong>Register</strong> with <a href="https://forms.gle/NHSTSwgbCDdQzuxKA">Google Forms</a></p>
<p>💥 <strong>Registration Fee</strong>: 5100 BDT (for students),8160BDT (for professionals)</p>
<p>📝 To register for the workshop, follow instructions in the email “<em>Weekend Workshops</em>” you received after registration.</p>
<hr>
<section id="overview" class="level1">
<h1>Overview</h1>
<p>This course is designed to introduce students to the use of R/Bioconductor for bioinformatics data analysis. R/Bioconductor is a popular open-source programming language and environment used for statistical computing and graphics, specifically for the analysis of biological data. The course will cover the basics of R programming language and its applications to bioinformatics, including the use of Bioconductor packages for analysis of high-throughput sequencing data, gene expression data, and proteomics data.</p>
<p>Students will learn how to perform exploratory data analysis, visualization, quality control, and statistical analysis using R/Bioconductor packages. Topics covered will include data import and export, data manipulation and transformation, normalization, differential expression analysis, functional enrichment analysis, and pathway analysis. The course will also provide an introduction to machine learning techniques for bioinformatics analysis.</p>
<p>Throughout the course, students will apply their knowledge through practical exercises and projects. By the end of the course, students will have a good understanding of R/Bioconductor and be able to apply it to their own bioinformatics research projects.</p>
<section id="what-is-r" class="level2">
<h2 class="anchored" data-anchor-id="what-is-r">What is R?</h2>
<p>R is an open source language and environment for statistical computing, data mining, modeling, and data graphics. It provides a wide variety of statistical and graphical techniques such as linear and non-linear modeling, statistical tests, time series analysis, classification, and clustering.</p>
</section>
<section id="what-is-bioconductor" class="level2">
<h2 class="anchored" data-anchor-id="what-is-bioconductor">What is Bioconductor?</h2>
<p>Bioconductor is an open-source, open-development software project for the analysis and comprehension of genomic data, primarily focused on the R programming language. Bioconductor provides a wide range of tools for the analysis and interpretation of high-throughput genomic data, including microarrays, next-generation sequencing data, proteomics data, and other genomic data types.</p>
<p>Bioconductor consists of a collection of packages that are designed for specific genomic data analysis tasks, such as normalization, differential gene expression analysis, pathway analysis, visualization, and machine learning. The packages are regularly updated and maintained by a large and active community of developers and users.</p>
<p>Bioconductor also provides an extensive documentation, including tutorials, vignettes, and manuals, to help users get started with the software and to guide them through the analysis process.</p>
<p>In summary, Bioconductor is an essential resource for researchers and scientists working in the field of genomics and bioinformatics, providing a powerful and flexible platform for the analysis and interpretation of genomic data.</p>
</section>
<section id="learning-objectives" class="level2">
<h2 class="anchored" data-anchor-id="learning-objectives">Learning Objectives</h2>
<p>Upon completion of a course on R/Bioconductor for Bioinformatics, students will be able to:</p>
<p><strong>1. Understand the basics of R programming language and Bioconductor packages for bioinformatics</strong></p>
<ul>
<li>Understand the syntax, data types, control structures, and functions in R</li>
<li>Understand the Bioconductor infrastructure, including the package installation, management, and documentation</li>
</ul>
<p><strong>2. Perform exploratory data analysis, visualization, quality control, and statistical analysis using R/Bioconductor</strong></p>
<ul>
<li>Load and manipulate different types of genomic data, including microarray, RNA-seq, ChIP-seq, and proteomics data</li>
<li>Perform exploratory data analysis and visualization using R/Bioconductor packages</li>
<li>Perform quality control of genomic data and identify problematic samples</li>
<li>Use statistical methods for hypothesis testing, such as t-tests, ANOVA, and regression analysis</li>
</ul>
<p><strong>3. Apply R/Bioconductor to high-throughput sequencing data, gene expression data, and proteomics data analysis</strong></p>
<ul>
<li>Process and normalize high-throughput sequencing data using R/Bioconductor packages</li>
<li>Perform differential expression analysis of RNA-seq data to identify differentially expressed genes</li>
<li>Perform ChIP-seq analysis to identify enriched regions and motifs</li>
<li>Perform proteomics data analysis to identify differentially expressed proteins</li>
<li>Perform differential expression analysis, functional enrichment analysis, and pathway analysis using R/Bioconductor</li>
</ul>
<p><strong>4. Perform differential expression analysis of genomic data to identify differentially expressed genes, transcripts, or proteins</strong></p>
<ul>
<li>Perform functional enrichment analysis to identify overrepresented biological pathways or gene ontology terms</li>
<li>Perform pathway analysis to identify pathways that are significantly altered in different biological conditions</li>
</ul>
<p><strong>5. Apply machine learning techniques for bioinformatics analysis</strong></p>
<ul>
<li>Understand the principles of machine learning and supervised and unsupervised learning</li>
<li>Apply machine learning techniques, such as random forests, support vector machines, and principal component analysis, for the analysis of genomic data</li>
</ul>
<p><strong>6. Design and implement a bioinformatics research project using R/Bioconductor</strong></p>
<ul>
<li>Understand the steps involved in designing and implementing a bioinformatics research project</li>
<li>Identify and formulate a research question that can be answered using genomic data analysis</li>
<li>Collect and preprocess the required genomic data</li>
<li>Perform the required genomic data analysis using R/Bioconductor packages</li>
<li>Interpret and visualize the results of genomic data analysis</li>
<li>Communicate the findings of genomic data analysis effectively in written and oral formats.</li>
</ul>
</section>
<section id="why-r" class="level2">
<h2 class="anchored" data-anchor-id="why-r">Why R?</h2>
<ul>
<li>R is a programming and statistical language.</li>
<li>R is used for data Analysis and Visualization.</li>
<li>R is simple and easy to learn, read and write.</li>
<li>R is an example of a FLOSS (Free Libre and Open Source Software) where one can freely distribute copies of this software, read its source code, modify it, etc.</li>
</ul>
</section>
<section id="useful-resources-for-learning-r" class="level2">
<h2 class="anchored" data-anchor-id="useful-resources-for-learning-r">Useful resources for learning R</h2>
<ul>
<li><p><strong>DataCamp:</strong> interactive online lessons in R.</p>
<ul>
<li>Some of the courses are free (particularly community-written lessons like the one you’ll do today), but for paid courses, DataCamp costs about 300 SEK / mo.</li>
</ul></li>
<li><p><strong>RStudio Cheat Sheets:</strong> Very helpful 1-2 page overviews of common tasks and packages in R.</p></li>
<li><p><strong>Quick-R:</strong> Website with short example-driven overviews of R functionality.</p></li>
<li><p><strong>StackOverflow:</strong> Part of the Stack Exchange network, StackOverflow is a Q&amp;A community website for people who work in programming. Tons of incredibly good R users and developers interact on StackExchange, so it’s a great place to search for answers to your questions.</p></li>
<li><p><strong>R-Bloggers:</strong> Blog aggregagator for posts about R. Great place to learn really cool things you can do in R.</p></li>
<li><p><strong>R for Data Science:</strong> Online version of the book by Hadley Wickham, who has written many of the best packages for R, including the Tidyverse, which we will cover.</p></li>
</ul>
</section>
<section id="textbook" class="level2">
<h2 class="anchored" data-anchor-id="textbook">Textbook</h2>
<ul>
<li><p>Wickham, H. &amp; Grolemund, G. (2018). for Data Science. O’Reilly: New York. Available for free at <a href="http://r4ds.had.co.nz" class="uri">http://r4ds.had.co.nz</a> (FREE)</p></li>
<li><p>Sosulski, K. (2018). R Fundamentals. Bookdown: New York. Available at: <a href="http://becomingvisual.com/rfundamentals" class="uri">http://becomingvisual.com/rfundamentals</a> (FREE)</p></li>
</ul>
</section>
<section id="required-software" class="level2">
<h2 class="anchored" data-anchor-id="required-software">Required software</h2>
<ul>
<li>R: <a href="http://www.r-project.org/" class="uri">http://www.r-project.org/</a> (FREE)</li>
<li>RStudio (additional libraries required): <a href="http://www.rstudio.com/" class="uri">http://www.rstudio.com/</a> (FREE)</li>
</ul>
</section>
<section id="recording-of-classes" class="level2">
<h2 class="anchored" data-anchor-id="recording-of-classes">Recording of classes</h2>
<p>Class lectures will be recorded automatically using cloud. The links will be posted to CHIRAL Classes when they are available.</p>
</section>
<section id="is-this-course-for-me" class="level2">
<h2 class="anchored" data-anchor-id="is-this-course-for-me">Is this course for me?</h2>
<p>If your answer to any of the following questions is “yes”, then this is the right workshop for you.</p>
<ul>
<li><p>Do you make summary tables in R (data, survey data, regression models, time-to-event data, adverse event reports)?</p></li>
<li><p>Do you want your workflow to be reproducible?</p></li>
<li><p>Are you often frustrated with the immense amount of code required to create great-looking tables in R?</p></li>
</ul>
<p>The workshop is designed for those with some experience in R. It will be expected that you can perform basic data manipulation. Experience with the {tidyverse} and the <code>%&gt;%</code> operator is a plus, but not required.</p>
</section>
<section id="prework" class="level2">
<h2 class="anchored" data-anchor-id="prework">Prework</h2>
<p>Before attending the workshop please have the following installed and configured on your machine.</p>
<ul>
<li><p>Recent version of <strong>R</strong></p></li>
<li><p>Recent version of <strong>RStudio</strong></p></li>
<li><p>Most recent release of the <a href="https://www.bioconductor.org/install/"><strong>Bioconductor</strong></a> and other packages used in courses</p></li>
</ul>
<p>Install the latest release of R, then get the latest version of Bioconductor by starting R and entering the commands.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> (<span class="sc">!</span><span class="fu">require</span>(<span class="st">"BiocManager"</span>, <span class="at">quietly =</span> <span class="cn">TRUE</span>))</span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a>    <span class="fu">install.packages</span>(<span class="st">"BiocManager"</span>)</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a>BiocManager<span class="sc">::</span><span class="fu">install</span>(<span class="at">version =</span> <span class="st">"3.16"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<ul>
<li><p>Ensure you can knit R markdown documents</p>
<ul>
<li>Open RStudio and create a new Rmarkdown document</li>
<li>Save the document and check you are able to knit it.</li>
</ul></li>
</ul>
</section>
<section id="install-bioconductor-packages" class="level2">
<h2 class="anchored" data-anchor-id="install-bioconductor-packages">Install Bioconductor Packages</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb2"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> (<span class="sc">!</span><span class="fu">require</span>(<span class="st">"BiocManager"</span>, <span class="at">quietly =</span> <span class="cn">TRUE</span>))</span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a>    <span class="fu">install.packages</span>(<span class="st">"BiocManager"</span>)</span>
<span id="cb2-3"><a href="#cb2-3" aria-hidden="true" tabindex="-1"></a>BiocManager<span class="sc">::</span><span class="fu">install</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>Install specific packages, e.g., “GenomicFeatures” and “AnnotationDbi”, with</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a>BiocManager<span class="sc">::</span><span class="fu">install</span>(<span class="fu">c</span>(<span class="st">"GenomicFeatures"</span>, <span class="st">"AnnotationDbi"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>The install() function (in the BiocManager package) has arguments that change its default behavior; type ?install for further help.</p>
</section>
<section id="zoom-working-virtually" class="level2">
<h2 class="anchored" data-anchor-id="zoom-working-virtually">Zoom + Working Virtually</h2>
<ul>
<li><p>Zoom link will be emailed to students</p></li>
<li><p>Class sessions will be recorded and later posted</p></li>
<li><p>We will have lectures as well as breakout room sessions to work on labs</p></li>
<li><p>Please be aware that there is the option to use closed captioning:</p></li>
</ul>
</section>
<section id="team" class="level2">
<h2 class="anchored" data-anchor-id="team">Team</h2>
<section id="md.-jubayer-hossain" class="level4">
<h4 class="anchored" data-anchor-id="md.-jubayer-hossain">Md. Jubayer Hossain</h4>
<p><img src="images/JHossain.jpg" class="img-fluid" style="float:right;padding: 0 0 0 10px;" alt="Headshot of Jubayer" width="200"></p>
<p><a href="">Md. Jubayer Hossain</a> is the Founder and Data Analyst at <a href="">CHIRAL Bangladesh</a>. CHIRAL Bangladesh is a non-profit organization dedicated to health research to improve lives in Bangladesh. He aspires to maximize the quality of life of the human around me by working at the intersection of education, technology, and health research. His research interests include health data science and machine learning in healthcare.</p>
<p><br></p>
</section>
<section id="md.-wahidul-islam" class="level4">
<h4 class="anchored" data-anchor-id="md.-wahidul-islam">Md. Wahidul Islam</h4>
<p><img src="images/wahidul.jpg" class="img-fluid" style="float:right;padding: 0 0 0 10px;" alt="Headshot of Wahidul" width="200"></p>
<p>Wahidul Islam is an undergraduate student in the Department of Microbiology at Jagannath University, Dhaka, Bangladesh. Bioinformatics, Immunology, Cell Biology, Biostatistics, Public health, and Microbiology are some of his research interests. Currently, he is working as a research assistant in Population Health Studies Division &amp; Genomic Data Science Center at CHIRAL Bangladesh. He also leads the seminar and workshop team at CHIRAL Bangladesh.</p>
<p><br></p>
</section>
<section id="muhibullah-shahjahan" class="level4">
<h4 class="anchored" data-anchor-id="muhibullah-shahjahan">Muhibullah Shahjahan</h4>
<p><img src="images/muhibullah.jpg" class="img-fluid" style="float:right;padding: 0 0 0 10px;" alt="Headshot of muhibullah" width="200"></p>
<p>Muhibullah is an undergraduate student in the Department of Microbiology at Jagannath University and health data analysis, bioinformatics are some of his research interests. Currently, he is working as a research assistant at CHIRAL Bangladesh.</p>


</section>
</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
  </div>
  <div class="nav-page nav-page-next">
      <a href="./q-and-a.html" class="pagination-link">
        <span class="nav-page-text">FAQs</span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">This page is built with ❤️ and <a href="https://quarto.org/">Quarto</a>.</div>   
    <div class="nav-footer-right">© Copyright 2022, Md. Jubayer Hossain</div>
  </div>
</footer>



</body></html>